homeassistant/components/palazzetti/strings.json

```diff
@@ -8,6 +8,10 @@
         "data_description": {
           "host": "The host name or the IP address of the Palazzetti CBox"
         }
+      },
+      "discovery_confirm": {
+        "title": "Discovered Palazzetti device",
```

Comment:
Small tweak, but we do not need to add this title or translation, as the user already gets to this screen from discovery, it is already clear this was discovered.

```suggestion
```